### ISSUE TITLE
node.js module "sys" renamed to "util"

### ISSUE THREAD

**Issue by meijeru** [giesse/Project-SnowBall#1]
When compiling `bootstrap.js` with node (version 0.6.6 for Windows) I get the informative message:
The "sys" module is now called "util". It should have a similar interface.

I have produced a new interpreter (`topaz.js`) by changing the line

```
    sys: apply require ["sys"]
```

into

```
    sys: apply require ["util"]
```

in `compile-topaz.topaz` 

This then does not give the informative message any more, and seems to work OK

**Comment by giesse** [comment]
Yes, that should be enough to fix it. I've been using an old version of node here, but I have updated my system to LMDE the other day, so I'll probably switch to the latest node as well.

Sorry for these kind of issues, as soon as the "real" compiler will be ready, things like this one will be abstracted out in a better way.

**Comment by giesse** [comment]
Fixed on 3ea5fed813282298bfb9376485200617a6122d73
